README.md

```diff
@@ -119,7 +119,15 @@ github-linguist.gemspec
 lib/linguist.rb
 …
 ```
-
+## Docker
+If you have docker installed you can build an image and run this as a container.
+```
+docker build -t my-linguist .
```

Comment:
I think this should just be `linguist` in the example. It's not like their implementing their own child version of Linguist.
```suggestion
docker build -t linguist .
```